.github/workflows/build-rpm.yml

```diff
@@ -133,9 +138,6 @@ jobs:
           ls -al
           curl -u ${{ secrets.JFROG_USERNAME }}:"${{secrets.JFROG_PASSWORD}}" -X PUT "https://genesisglobal.jfrog.io/artifactory/${{ inputs.artifactory_deploy_locations }}" -T $HOME/rpmbuild/RPMS/noarch/*.rpm
 
-      # - name: build the RPM
-      #   run: |
-      #     rpmbuild -bs
-
-
-
+      - name: Cleanup
+        run: |
+          rm -rf $HOME/rpmbuild/
```

Comment:
We can get rid of all of these lines

Reply:
lines has been  removed from rpmtest branch